scc-hypervisor-collector/xml/art-scc-hypervisor-collector.xml

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+  <!ENTITY productnamex "<phrase xmlns='http://docbook.org/ns/docbook' os='sles;sled;osuse'>&productname;</phrase><phrase xmlns='http://docbook.org/ns/docbook' os='article'>&sls;</phrase>">
+   <!ENTITY productnumberx "<phrase xmlns='http://docbook.org/ns/docbook' os='sles;sled;osuse'>&productnumber;</phrase>">
+]>
+<article xml:id="article-scc-hypervisor-collector"
+         xmlns="http://docbook.org/ns/docbook"
+         version="5.0"
+         xml:lang="en"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>Hypervisor details in &scc;</title>
+  <titleabbrev>Hypervisor details in &scc;</titleabbrev>
+  <info><productname>&productnamex;</productname>
+    <productnumber>&productnumberx;</productnumber><date>
+<?dbtimestamp format="B d, Y" ?></date>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker>
+        <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
+        <dm:component>Documentation</dm:component>
+        <dm:assignee>tbazant@suse.com</dm:assignee>
+      </dm:bugtracker>
+      <dm:editurl>https://github.com/SUSE/doc-unversioned/blob/main/xml/</dm:editurl>
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+    <abstract>
+      <para>
+        This article describes how to provide customers with details about the
+        hypervisors they use. The collected data is available under the
+        customers' &scc; (SCC) account.
+      </para>
+    </abstract>
+  </info>
+  <sect1 xml:id="scc-hypervisor-collector-whatis">
+    <title>What is &scchvc;?</title>
+
+    <para>
+      &scchvc; is a command line tool that collects details about customer
+      hypervisor solutions and uploads them to SCC.
+    </para>
+  </sect1>
+  <sect1 xml:id="scc-hypervisor-collector-requirements">
+    <title>Requirements</title>
+
+    <para>
+      To collect hypervisor data and upload them to SCC, you need to:
```

Comment:
The "tone" of this section may not "fit" with the original intent of the scc-hypervisor-collector...

The original goal was to provide a tool that customers could easily use without necessarily having to setup a special host to run it, hence the focus on providing a container image as well that can be run from any host that supports running containers with docker or podman...

So perhaps the phrasing here should be more along the lines of:

The easiest way to run scc-hypervisor-collector is on an existing registered SLE-based host...